src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/WindowsUpdateManager.cs

```diff
@@ -341,7 +359,7 @@ private bool HandleWUOperationStates(CancellationToken cancellationToken)
                             {
                                 string msg =
                                     "Current Operation state : InstallationApproved but no updates found to install. Completing the operation.";
```

Comment:
>Current Operation state : [](start = 37, length = 25)

The format of this isnt same as other message. Should you remove Current Operation state?